.toolchain.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Hi, there!
+# This is passed via jenkins podTemplate argument.
+# Instead of trying injections, take a look at https://en.pingcap.com/careers/.
+image_tag_suffix: -llvmorg-14.0.6
```

Comment:
Seems that it didn't apply to the build process? https://ci.pingcap.net/blue/organizations/jenkins/tiflash-build-common/detail/tiflash-build-common/7534/pipeline/155
```
[2022-07-01T02:41:18.259Z] + cmake /home/jenkins/agent/workspace/tiflash-build-common/tiflash -DPREBUILT_LIBS_ROOT=/home/jenkins/agent/workspace/tiflash-build-common/tiflash/contrib/tiflash-proxy/ -G Ninja -DENABLE_FAILPOINTS=true -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_INSTALL_PREFIX=/home/jenkins/agent/workspace/tiflash-build-common/install/tiflash -DENABLE_TESTS=false -DUSE_CCACHE=true -DDEBUG_WITHOUT_DEBUG_INFO=true -DUSE_INTERNAL_TIFLASH_PROXY=false -DRUN_HAVE_STD_REGEX=0
[2022-07-01T02:41:18.259Z] -- The C compiler identification is Clang 13.0.0
[2022-07-01T02:41:18.517Z] -- The CXX compiler identification is Clang 13.0.0
```